content/solutions/java.html.haml

```diff
@@ -37,3 +37,39 @@ plugins:
   - 'Repository Connector plugin'
   - 'adds features for resolving artifacts from a Maven repository such as Nexus or Artifactory.'
 ---
+
+:markdown
+Jenkins supports building Java projects since its inception, and for a reason!
```

Comment:
The content under a `:markdown` declaration should be indented two spaces. HAML is whitespace sensitive